main/src/cgeo/geocaching/CacheListActivity.java

```diff
@@ -101,6 +102,7 @@
 import android.widget.AdapterView.AdapterContextMenuInfo;
 import android.widget.ListView;
 import android.widget.TextView;
+import android.widget.Toast;
```

Comment:
`Toast` is not used anywhere in the rest of the changes to this file (while all the changes are seen together). It must be wrong.